packages/app/background-jobs-common/test/dependencyMocks.ts

```diff
@@ -31,7 +31,7 @@ export class DependencyMocks {
   }
 
   async dispose(): Promise<void> {
-    await this.client?.quit()
+    await this.client?.disconnect(false)
```

Comment:
Using `.quit()` is discouraged in favor of `.disconnect()`. 
`QUIT` is treated as a command, which could potentially delay connection closure.
This is a test file so shouldn't impact the functionality of the lib.